src/nnfusion/engine/pass/graph/hlsl_required_pass.cpp

```diff
@@ -92,10 +92,7 @@ namespace
 
 bool HLSLRequiredPass::run_on_graph(std::shared_ptr<nnfusion::graph::Graph>& graph)
 {
-    if (FLAGS_fdefault_device != "HLSL")
-    {
-        return true;
-    }
+    return true;
 
```

Comment:
Why do we need to disable this pass? And if necessary, un-register this pass in engine may be better.